common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -96,7 +92,10 @@ public void onContainerRender(ContainerRenderEvent event) {
                 case TEXTURE -> {
                     int adjustedY;
                     if (displayBottom) {
-                        int displayedTextures = getFilteredEmeraldAmounts(topEmeralds).length;
+                        int displayedTextures = Arrays.stream(getRenderableEmeraldAmounts(topEmeralds))
+                                .filter(s -> !s.equals("0"))
```

Comment:
what about `!showZerosInEmeraldCount.get()`?

Reply:
Didn't check because I only looked at the container in the issue, fixed now